Tests/tbDEXTests/HttpClient/tbDEXHttpClientTests.swift

```diff
@@ -125,15 +126,16 @@ final class tbDEXHttpClientTests: XCTestCase {
     }
     
     func test_getBalancesWithOneValidBalance() async throws {
-        let response = validBalance
+        var balance = DevTools.createBalance(from: altDid.uri)
+        try balance.sign(did: altDid)
         
         Mocker.mode = .optin
         Mock(
             url: URL(string: "\(endpoint)/balances")!,
             contentType: .json,
             statusCode: 200,
             data: [
-                .get: response.data(using: .utf8)!
+                .get: try tbDEXJSONEncoder().encode(["data": [balance]])
```

Comment:
just want to make sure coz this confused me tooo - are you assuming balance objects are in an array in 1 tbdex balance message?

Reply:
actually, i don't think you are - this is the http response that returns
```
{
   "data": [1 tbdex message of type balance, with metadata data signature ]
}

```

so nvm!